docs-app/package.json

```diff
@@ -0,0 +1,129 @@
+    "@glint/environment-ember-template-imports": "^0.9.6",
+    "@glint/template": "^0.9.6",
+    "@nullvoxpopuli/eslint-configs": "3.0.2",
+    "@tailwindcss/typography": "^0.5.7",
+    "@tsconfig/ember": "^2.0.0",
+    "@types/ember": "^4.0.1",
+    "@types/ember-qunit": "^5.0.2",
+    "@types/ember-resolver": "^5.0.11",
+    "@types/ember__application": "^4.0.2",
+    "@types/ember__array": "^4.0.2",
+    "@types/ember__component": "^4.0.10",
+    "@types/ember__controller": "^4.0.1",
+    "@types/ember__debug": "^4.0.1",
+    "@types/ember__destroyable": "^4.0.0",
+    "@types/ember__engine": "^4.0.2",
+    "@types/ember__error": "^4.0.0",
+    "@types/ember__object": "^4.0.4",
+    "@types/ember__owner": "^4.0.0",
+    "@types/ember__polyfills": "^4.0.0",
+    "@types/ember__routing": "^4.0.11",
+    "@types/ember__runloop": "^4.0.1",
+    "@types/ember__service": "^4.0.0",
+    "@types/ember__string": "^3.0.9",
+    "@types/ember__template": "^4.0.0",
+    "@types/ember__test": "^4.0.0",
+    "@types/ember__utils": "^4.0.1",
+    "@types/qunit": "^2.19.3",
+    "@types/rsvp": "^4.0.4",
+    "autoprefixer": "^10.4.12",
+    "concurrently": "^7.4.0",
+    "cssnano": "^5.1.13",
+    "ember-auto-import": "^2.4.2",
+    "ember-cli": "~4.10.0",
+    "ember-cli-app-version": "^6.0.0",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-deprecation-workflow": "^2.1.0",
+    "ember-cli-htmlbars": "^6.1.1",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-fetch": "^8.1.2",
+    "ember-load-initializers": "^2.1.2",
+    "ember-page-title": "^8.0.0-beta.0",
+    "ember-qunit": "^6.0.0",
+    "ember-resolver": "^10.0.0",
+    "ember-source": "~4.10.0",
+    "ember-template-imports": "^3.1.2",
+    "ember-template-lint": "^4.14.0",
+    "eslint": "^8.32.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-ember": "^11.4.3",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-qunit": "^7.3.1",
+    "loader.js": "^4.7.0",
+    "postcss": "^8.4.17",
+    "postcss-import": "^15.0.0",
+    "postcss-loader": "^7.0.1",
+    "prettier": "^2.8.3",
+    "prettier-plugin-ember-template-tag": "^0.3.0",
+    "qunit": "^2.19.1",
+    "qunit-dom": "^2.0.0",
+    "tailwindcss": "^3.1.8",
+    "typescript": "^4.8.4",
+    "webpack": "^5.74.0"
+  },
+  "engines": {
+    "node": "18.* || >= 20"
+  },
+  "ember": {
+    "edition": "octane"
+  },
+  "dependencies": {
+    "@crowdstrike/ember-oss-docs": "^1.1.3",
+    "@ember/test-waiters": "^3.0.2",
+    "@embroider/router": "^2.0.0",
+    "ember-browser-services": "^4.0.4",
+    "ember-cached-decorator-polyfill": "^1.0.1",
+    "ember-headless-form": "workspace:*",
+    "ember-modifier": "^4.0.0",
+    "ember-resources": "^5.4.0",
+    "highlight.js": "^11.6.0",
+    "highlightjs-glimmer": "^1.4.1",
+    "tracked-built-ins": "^3.1.0"
+  }
```

Comment:
Do you need the `dependenciesMeta` here as well? E.g.,:

```
"dependenciesMeta": {
    "ember-headless-form": {
      "injected": true
    }
  }
```

(https://github.com/CrowdStrike/ember-toucan-core/blob/main/docs-app/package.json#L134)

Reply:
It seems like it was working without it based on what I read from a previous PR (let me dig it up)...but then how come we need it in other packages?? (don't feel the need to answer this if it's going to take a lot of investigation btw, just curious)

Reply:
Update, yeah you are right it was added over here: https://github.com/CrowdStrike/ember-headless-form/pull/49/files#diff-b03f4acb30e04ceb12a40763ef649759b3f632df2357bcec14fab6dd29ebbfb4R124 in the test app

Reply:
It wasn't needed as I wasn't really using the addon in the docs app yet. But yes, your are right, we should do that here also - and I just did that, to prevent any more foot guns due to this unfortunate issue.